src/coretoolbox.rs

```diff
@@ -533,6 +533,9 @@ mod entrypoint {
         // Allow sudo
         || -> Fallible<()> {
             let f = std::fs::File::create(format!("/etc/sudoers.d/toolbox-{}", state.username))?;
+            let mut perms = f.metadata()?.permissions();
```

Comment:
I thought this wouldnt work, because we then write to it below... but I tested this and it does work.